users/models.py

```diff
@@ -1,23 +1,26 @@
 from django.db import models
 from django.contrib.auth import get_user_model
 from utils.constants import BILLING_ADDRESS, SHIPPING_ADDRESS
+from final_project.models import CustomModel
 
 AuthUserModel = get_user_model()
-
+"This represents the usermodels, user model default from django added in a variable"
```

Comment:
This is still a string data ... why are you throwing a string data randomly in your code?

Reply:
I meant to put ''' sorry :D